azerothcore/azerothcore-wotlk
# fix(DB/quest): Thalorien Dawnseeker - Quest texts

## CHANGES PROPOSED:
- Add quest texts for quest 24535 & 24563
- Add locale texts for deDE

**Quest 24535**
![2020-12-06 10_27_25-World of Warcraft](https://user-images.githubusercontent.com/55922592/101276663-c9897c80-37ae-11eb-9472-228d0575bbb6.png)

**Quest 24535**
![2020-12-06 10_14_22-World of Warcraft](https://user-images.githubusercontent.com/55922592/101276664-cbebd680-37ae-11eb-8086-a4d6c9936ffe.png)

**Quest 24563**
![2020-12-06 10_14_59-World of Warcraft](https://user-images.githubusercontent.com/55922592/101276666-cdb59a00-37ae-11eb-84d6-0331a6410fe1.png)


Source: 
- https://youtu.be/UFURxcDcTlQ?t=329
- https://www.wowhead.com/quest=24535/thalorien-dawnseeker
- https://db.rising-gods.de/?quest=24535
- https://db.rising-gods.de/?quest=24563

## ISSUES ADDRESSED:
- https://github.com/azerothcore/azerothcore-wotlk/issues/3851



## TESTS PERFORMED:
- Applied SQL script
- Tested ingame



## HOW TO TEST THE CHANGES:


1. .go c 43498
2. .q rem 24535, .q a 24535 & .q c 24535
3. Check quest texts 
4. .qrem 24563, .q a 24563 & q.c 24563
5. Check quest texts again

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
